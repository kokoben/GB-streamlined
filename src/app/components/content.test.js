import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Content from './content';

it('renders correctly', () => {
  const wrapper = shallow(<Content />);
  const tree = toJson(wrapper);
  expect(tree).toMatchSnapshot();
});
