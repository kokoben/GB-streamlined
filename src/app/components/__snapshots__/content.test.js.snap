// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Adapter
  className="content"
  style={
    Object {
      "background": "#fff",
      "borderBottomColor": "#981616",
      "borderBottomStyle": "solid",
      "borderBottomWidth": "2px",
      "margin": "0 auto 50px",
      "padding": "10px 10px",
    }
  }
>
  <main>
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/quicklooks/"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/features"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/bombcasts"
      />
    </Switch>
  </main>
</Adapter>
`;
